test/UniformlyRefinedForestOfOctreesDiscreteModelsTests.jl

```diff
@@ -27,70 +27,73 @@ module UniformlyRefinedForestOfOctreesDiscreteModelsTests
   end
 
   function run(parts,subdomains,num_uniform_refinements)
-    # Manufactured solution
-    u(x) = x[1] + x[2]
-    f(x) = -Δ(u)(x)
+    GridapP4est.with(parts;p4est_verbosity_level=P4est_wrapper.SC_LP_STATISTICS) do 
+        # Manufactured solution
+        u(x) = x[1] + x[2]
+        f(x) = -Δ(u)(x)
 
-    if length(subdomains)==2
-      domain=(0,1,0,1)
-    else
-      @assert length(subdomains)==3
-      domain=(0,1,0,1,0,1)
-    end
+        if length(subdomains)==2
```

Comment:
it seems you are using some kind of auto-identation feature in VScode ... is that the case? 

There are plently of changes here which are not actually code changes (just identation changes) and it gets confusing to do the review.

Perhaps you can let me know how you have configured the environment so that we can be both on the same page in regards to identation?